inhouse_bot/cogs/stats_cog.py

```diff
@@ -293,3 +212,74 @@ async def mmr_history(self, ctx: commands.Context):
             temp.close()
 
     # TODO MEDIUM PRIO (simple) Add !champions_stats once again!!!
+
+    @commands.command(aliases=["scout", "profile"])
+    @guild_only()
+    @doc(
+        f"""
+        Displays information about a player and champs used in inhouses
+        Example:
+            {PREFIX}scout @User
+            {PREFIX}profile @User
+    """
+    )
+    async def scouting(self, ctx: commands.Context, user: discord.Member):
+        with session_scope() as session:
+            player = (
+                session.query(Player)
+                .select_from(Player)
+                .filter(Player.server_id == ctx.guild.id)
+                .filter(Player.id == user.id)
+            ).one_or_none()
+
+            if not player:
+                await ctx.send("This player has no games played yet.")
+                return
+
+            # #### Create the initial embed object ####
+
+            # All Users default to unverified until further data is processed
+            embed = discord.Embed()
+            file = discord.File("riot-ranks/UNVERIFIED.png")
+            embed.set_thumbnail(url="attachment://UNVERIFIED.png")
+
+            if player.summoner_puuid:
+                summoner = await lol.get_summoner_by_puuid(player.summoner_puuid)
+                playerRankInfo = await lol.get_summoner_rank_info_by_id(summoner.id)
+                url = "https://www.op.gg/summoners/na/" f"{summoner.name}"
+                url = url.replace(" ", "")
+
+                # If a player has a rank whether it being (Solo/Duo or Flex or TFT)
+                if playerRankInfo:
+                    embed.set_author(
```

Comment:
Depending on the look you're going for, you can set the embed.title and embed.url instead of the author to have a more obvious hyperlink